project.clj

```diff
@@ -1,8 +1,4 @@
-(def clj-version "1.7.0")
-(def tk-version "1.5.0")
-(def tk-jetty-version "1.5.10")
-(def ks-version "1.3.1")
-(def ps-version "2.6.1-stable-SNAPSHOT")
+(def ps-version "2.6.1-master-SNAPSHOT")
```

Comment:
I think you do want to change this to 'stable', regardless of whether we do anything with the numbers yet.